Khan/wonder-blocks
# Make `layout` a required prop on Banner

## Summary:
- Updated stories to show that devs should manually add spacing around floating banners, it is _not_ part of the banner itself.
- Made `layout` a required prop with no default.

I actually got rid of the "Variants" story because it
starting looking really awkward and I didn't think it was
worth updating. Instead, I added a specific "Kinds" story
and a specific "Layouts" story.

Updates to come in future PRs:
- Updating the icon to be an Icon with IconAsset instead of an img
- Changing the buttons to be small (not adding this in the current PR so that I can keep my branches consistent)
- Comprehensive accessibility features and testing
- This includes role and aria-live

**Storybook publish:**
https://5e1bf4b385e3fb0020b7073c-jannpabsfz.chromatic.com/?path=/docs/banner--default

**Chromatic:**
https://www.chromatic.com/pullrequest?appId=5e1bf4b385e3fb0020b7073c&number=1617

Issue: https://khanacademy.atlassian.net/browse/WB-1385

## Test plan:
- Check the stories at http://localhost:6061/?path=/docs/banner--layouts
- Make sure the descriptions make sense and have no typos.
- Confirm that the stories look correct.
